src/test/java/org/entando/plugins/pda/pam/service/process/KieProcessServiceTest.java

```diff
@@ -39,6 +39,7 @@
 import org.springframework.test.web.client.MockRestServiceServer;
 import org.springframework.web.client.RestTemplate;
 
+@SuppressWarnings("PMD.ExcessiveImports")
```

Comment:
Not applicable.
If you didn't change this Class, why do you need to ignore this error?